azure-iot-hub/azure/iot/hub/connection_string.py

```diff
@@ -0,0 +1,93 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module contains tools for working with Connection Strings"""
```

Comment:
>module contains to [](start = 8, length = 18)

A lot of this code looks similar to connection string class we have in device.  would we be shifting these files (connection string and sas token) for common usage having copies in both packages ?

Reply:
Yes, we would at some point. We discussed this with Carter and we just keep a duplicate here for now.

---
In reply to: [324402207](https://github.com/Azure/azure-iot-sdk-python/pull/300#discussion_r324402207) [](ancestors = 324402207)